examples/README.md

```diff
@@ -15,6 +15,7 @@
 - [overfitting](overfitting): Notebook on underfitting, overfitting, regularization, dropout.
 - [resnet](resnet): Knet implementation of [ResNet] 50, 101, and 152 models.
 - [vgg](vgg): Knet implementation of [VGG] D and E models.
+- [bilstm-taggers](bilstm-taggers) Two different BiLSTM taggers on [WikiNER] data. [One](bilstm-taggers/bilstm-tagger.jl) has the word UNK in its vocab, [the other one](bilstm-taggers/bilstm-tagger-withchar.jl) generates UNK words' embeddings by using an another BiLSTM network which takes characters as input.
```

Comment:
Should add a single line for dynet-benchmarks here instead.

Reply:
OK, I'll do it, this is a branch merge mess.